.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Gradle Plugin Publish
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish plugin to gradle plugins portal
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: |
+            8
+            17
```

Comment:
Q: Do we need two Java versions to release?

Reply:
I tried to use 
```
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(8)
    }
}
```
in build.gradle. So putting 17 in last make us running gradle using JDK 17 and compile code using 8.
To speed up I make Actions download 8 first and above setting can detect installed 8 to avoid downloading by Gradle toolchain.

Reply:
I see.